ui/app/AppLayouts/Chat/ChatColumn/ChatButtons.qml

```diff
@@ -15,16 +15,23 @@ Item {
 
     Button {
         id: chatSendBtn
-        visible: txtData.length > 0
+        visible: txtData.length > 0 || chatColumn.isImage
         width: 30
         height: 30
         text: ""
         anchors.rightMargin: Style.current.padding
         anchors.verticalCenter: parent.verticalCenter
         anchors.right: parent.right
         onClicked: {
-            chatsModel.sendMessage(txtData.text, chatColumn.isReply ? SelectedMessage.messageId : "")
-            txtData.text = ""
+            if(chatColumn.isImage){
+                chatsModel.sendImage(sendImageArea.image);
+                chatColumn.hideExtendedArea();
```

Comment:
Do you need a return here or it's possible to send an image plus text?

Reply:
It is possible. They are sent as separate messages